unlock-app/src/components/interface/buttons/Button.js

```diff
@@ -1,71 +1,75 @@
-import { Link } from 'react-router-dom'
 import PropTypes from 'prop-types'
 import React from 'react'
+import Link from 'next/link'
 import styled from 'styled-components'
 
-export const LayoutButton = ({ to = '#', title, children }) => (
-  <Button
-    to={to}
-    backgroundcolor={'var(--grey)'}
-    fillcolor={'white'}
-    backgroundhovercolor={'var(--link)'}
-    fillhovercolor={'white'}
-  >
-    {children}
-    <Label>{title}</Label>
-  </Button>
+export const LayoutButton = ({href, title, children}) => (
+  <Link href={href}>
+    <Button
+      href={href}
+      backgroundColor={'var(--grey)'}
+      fillColor={'white'}
+      backgroundHoverColor={'var(--link)'}
+      fillHoverColor={'white'}
+    >
+      {children}
+      <Label>{title}</Label>
+    </Button>
+  </Link>
 )
 
 LayoutButton.propTypes = {
-  to: PropTypes.string,
+  href: PropTypes.string,
   title: PropTypes.string,
   children: PropTypes.node,
-  backgroundcolor: PropTypes.string,
-  backgroundhovercolor: PropTypes.string,
-  fillcolor: PropTypes.string,
-  fillhovercolor: PropTypes.string,
+  backgroundColor: PropTypes.string,
+  backgroundHoverColor: PropTypes.string,
+  fillColor: PropTypes.string,
+  fillHoverColor: PropTypes.string,
 }
 
-export const LockButton = ({ to = '#', children }) => (
-  <Button
-    to={to}
-    backgroundcolor={'var(--lightgrey)'}
-    fillcolor={'var(--grey)'}
-    backgroundhovercolor={'var(--link)'}
-    fillhovercolor={'white'}
-  >
-    {children}
-  </Button>
+export const LockButton = ({ href, children }) => (
+  <Link href={href}>
+    <Button
+      href={href}
+      backgroundColor={'var(--lightgrey)'}
+      fillColor={'var(--grey)'}
+      backgroundHoverColor={'var(--link)'}
+      fillHoverColor={'white'}
+    >
+      {children}
+    </Button>
+  </Link>
 )
 
 LockButton.propTypes = {
-  to: PropTypes.string,
+  href: PropTypes.string,
   children: PropTypes.node,
-  backgroundcolor: PropTypes.string,
-  backgroundhovercolor: PropTypes.string,
-  fillcolor: PropTypes.string,
-  fillhovercolor: PropTypes.string,
+  backgroundColor: PropTypes.string,
+  backgroundHoverColor: PropTypes.string,
+  fillColor: PropTypes.string,
+  fillHoverColor: PropTypes.string,
 }
 
-export const Button = styled(Link)`
-  background-color: ${props => props.backgroundcolor || 'var(--grey)'};
+export const Button = styled.a`
```

Comment:
Since you're using regular HTML `<a>` elements I worry that this will translate into regular links which are not being used by next to do client side nav?

Reply:
I had the same worry but, on reading more about Next.js, I think this is pretty neat. I really like the way it falls back - great for SEO, great for low page sizes and intelligent routing.

Reply:
@benwerd Correct! more on that [here](https://github.com/zeit/next.js#with-link).